src/main/java/org/broadinstitute/consent/http/models/Dataset.java

```diff
@@ -49,11 +49,8 @@ public class Dataset {
 
     private Boolean deletable;
 
-    private String sharingPlanDocument;
-
-    private String sharingPlanDocumentName;
-
```

Comment:
Correct me if I am wrong here, but I didn't see any usages of this field in consent or in the frontend. No datasets on dev had any value for this; in fact, since the value was null, no datasets on dev even had this field. Therefore, it seems safe to remove.